src/tinyYolov2/TinyYolov2.ts

```diff
@@ -205,12 +205,13 @@ export class TinyYolov2 extends NeuralNetwork<TinyYolov2NetParams> {
     for (let row = 0; row < numCells; row ++) {
       for (let col = 0; col < numCells; col ++) {
         for (let anchor = 0; anchor < numBoxes; anchor ++) {
-          const score = sigmoid(scoresTensor.get(row, col, anchor, 0))
+
+          const score = sigmoid(scoresTensor.arraySync()[row][col][anchor][0]);
           if (!scoreThreshold || score > scoreThreshold) {
-            const ctX = ((col + sigmoid(boxesTensor.get(row, col, anchor, 0))) / numCells) * correctionFactorX
-            const ctY = ((row + sigmoid(boxesTensor.get(row, col, anchor, 1))) / numCells) * correctionFactorY
-            const width = ((Math.exp(boxesTensor.get(row, col, anchor, 2)) * this.config.anchors[anchor].x) / numCells) * correctionFactorX
-            const height = ((Math.exp(boxesTensor.get(row, col, anchor, 3)) * this.config.anchors[anchor].y) / numCells) * correctionFactorY
+            const ctX = ((col + sigmoid(boxesTensor.arraySync()[row][col][anchor][0])) / numCells) * correctionFactorX
+            const ctY = ((row + sigmoid(boxesTensor.arraySync()[row][col][anchor][1])) / numCells) * correctionFactorY
+            const width = ((Math.exp(boxesTensor.arraySync()[row][col][anchor][2]) * this.config.anchors[anchor].x) / numCells) * correctionFactorX
+            const height = ((Math.exp(boxesTensor.arraySync()[row][col][anchor][3]) * this.config.anchors[anchor].y) / numCells) * correctionFactorY
 
```

Comment:
I am not sure if it's a good idea to call arraySync() on every iteration. I'd rather get the data once outside of the loop:

``` javascript 
  const scoresData = await scoresTensor.array()
  const boxesData = await boxesTensor.array()
```

Also you can turn this function into an async one.

Reply:
If you're right. I focused on direct migration and I did not realize those optimizations